src/sections/Meshery/Features-section/features-section.style.js

```diff
@@ -82,7 +86,7 @@ const FeaturesSectionWrapper = styled.section`
 			margin: 4rem 0;
 			align-items: center;
 		}
-		h2, h4 {
+		h2, h4, h3 {
```

Comment:
same case here in smp-section added h3 class to add styles but haven't used h3 in index.js file